### ISSUE TITLE
Autokey-Qt editing panel problems

### ISSUE THREAD

**Issue by DonaldSch** [autokey/autokey#435]
## Version: AutoKey version: 0.95.10

Used GUI: Qt 
Unknown if present in more than one version

Installed via:  Aur package using pacman

Linux Distribution: 
   Operating System: Manjaro Linux
   KDE Plasma Version: 5.19.3
   KDE Frameworks Version: 5.72.0
   Qt Version: 5.15.0
   Kernel Version: 5.4.52-1-MANJARO
   OS Type: 64-bit
   Processors: 4 × Intel® Core™ i5-2500K CPU @ 3.30GHz
   Memory: 15.6 GiB of RAM
   Graphics Processor: GeForce GTX 1060 3GB/PCIe/SSE2

## Summary
Spacing shown in scripts is not the same as shown when using Autokey-gtk and editing is not in sync with the cursor

## Steps to Reproduce

If text is entered or deleted the changes are at at different location to the cursor

## Expected Results

Editing and spacing to work the same as in Autokey-gtk

## Actual Results

- in this screenshot the cursor isn't captured but it is after the quotes at the end of "something else"
![image](https://user-images.githubusercontent.com/44590215/89367290-bdbc7980-d72c-11ea-9074-b1b56d70d1a0.png)
this is what happens when backspace is pressed
![image](https://user-images.githubusercontent.com/44590215/89367833-28ba8000-d72e-11ea-9b03-5a76762e9ad4.png)
the e of else is deleted when you expect the quotes to be deleted and the cursor is still showing after the qoutes at the end of "something lse"

##log during the above operation
```
Raw key: <Key.BACKSPACE: '<backspace>'>, modifiers: [], Key: Key.BACKSPACE
Window visible title: 'AutoKey', Window class: 'autokey-qt.autokey-qt'
Undoing last phrase expansion requested. Can undo last expansion: False
```

## Notes
Autokey-qtk works as expected for this but some icons don't show on my system
![image](https://user-images.githubusercontent.com/44590215/89369275-c06d9d80-d731-11ea-8a36-5eae8f912fa2.png)

**Comment by josephj11** [comment]
@DonaldSch Is this still a problem in 0.96.0?

I wouldn't know because I use dark mode and in autokey-qt I can't find the invisible cursor to start with. :)

I almost never use the editing panel. I use kate.

The log message indicates that AutoKey thought you invoked a phrase with an abbreviation and then pressed backspace - which undoes the expansion (you don't mention invoking a phrase). AutoKey phrases and scripts work on AutoKey windows too, so it is something that could happen.

**Comment by Elliria** [comment]
Although I currently experience this issue in **AutoKey 0.95.10** (which is what ships with Kubuntu 22.04 LTS), and it's a horrendous issue that renders the Qt editor completely useless, it was fixed in [AutoKey Qt 0.96.0-beta.10](https://github.com/autokey/autokey/releases/tag/v0.96.0-beta.10) and is working perfectly for me in [AutoKey 0.96.0](https://github.com/autokey/autokey/releases/tag/v0.96.0).

This issue was also mentioned in issue #474.

By the way, even though this issue is likely going to be closed, I thought I'd add that it should be renamed to take "**phrase**" out of the title to make searches more accurate, because it's an editor issue that affects phrases and scripts rather than just phrases.

**Comment by Elliria** [comment]
Whoops. Can we make that **Autokey-Qt editor problems** instead of **Autokey-Qt phrase problems**? That way it will be easier for others to find if they have a similar issue.
